tests/UI/campaigns/functional/API/01_basicTest.ts

```diff
@@ -0,0 +1,33 @@
+// Import utils
+import testContext from '@utils/testContext';
+
+import {expect} from 'chai';
+import {request, APIRequestContext} from '@playwright/test';
```

Comment:
You don't need to add '@playwright/test' the same types exists with playwright.

Reply:
Fixed